pyproject.toml

```diff
@@ -50,6 +50,8 @@ dependencies = [
     "importlib_metadata >=4.8.3 ; python_version<\"3.10\"",
     "jupyter-server >=1.0",
     "simpervisor >=1.0",
+    "tornado >=5.1",
```

Comment:
Does removing the tornado 4.5.x compatibility patch and setting the min version of tornado >=5.1 make this a breaking change? Trying to foresee which version of JSP this patch will be released with

Reply:
No, this is from #457, the PR is based on it for the moment.

Reply:
As long as we don't force a new major version in the dependency, i figure it should be fine at least.